python/pyspark/sql/dataframe.py

```diff
@@ -3526,8 +3526,8 @@ def head(self, n: Optional[int] = None) -> Union[Optional[Row], List[Row]]:
 
         Returns
         -------
-        If n is greater than 1, return a list of :class:`Row`.
-        If n is 1, return a single Row.
+        If n is supplied, return a list of :class:`Row`.
```

Comment:
```py
>>> spark.range(3).head(0)
[]
```
I'm wondering if we should call out the empty list when `n=0`.

Reply:
Good point. I'm suggesting adding this as an example, and extending the `returns` section to mention the length of the list (see commit below).

Reply:
Nice!